pinot-core/src/main/java/org/apache/pinot/core/query/aggregation/function/DistinctCountThetaSketchAggregationFunction.java

```diff
@@ -100,7 +101,7 @@ public DistinctCountThetaSketchAggregationFunction(List<ExpressionContext> argum
       ExpressionContext paramsExpression = arguments.get(1);
       Preconditions.checkArgument(paramsExpression.getType() == ExpressionContext.Type.LITERAL,
           "Second argument of DISTINCT_COUNT_THETA_SKETCH aggregation function must be literal (parameters)");
-      Parameters parameters = new Parameters(paramsExpression.getLiteral());
+      Parameters parameters = new Parameters(paramsExpression.getLiteralString());
```

Comment:
We can probably not have `getLiteralString()` ?

Reply:
See comment above